pkg/ebpf/events_pipeline.go

```diff
@@ -276,15 +346,20 @@ func (t *Tracee) processEvents(ctx context.Context, in <-chan *trace.Event) (<-c
 				continue
 			}
 
-			if (t.config.Filter.ContFilter.Value() || t.config.Filter.NewContFilter.Enabled()) && event.ContainerID == "" {
+			if t.config.FilterScopes.HasContainerFilterEnabled() && event.ContainerID == "" {
 				// Don't trace false container positives -
 				// a container filter is set by the user, but this event wasn't originated in a container.
 				// Although kernel filters shouldn't submit such events, we do this check to be on the safe side.
 				// For example, it might be that a new cgroup was created, and not by a container runtime,
 				// while we still didn't processed the cgroup_mkdir event and removed the cgroupid from the bpf container map.
-				id := events.ID(event.EventID)
-				// don't skip cgroup_mkdir and cgroup_rmdir so we can derive container_create and container_remove events
-				if id != events.CgroupMkdir && id != events.CgroupRmdir {
+				eventId := events.ID(event.EventID)
+
+				// don't skip cgroup_mkdir, cgroup_rmdir and security_sb_mount so we can derive container_create and container_remove events
+				if eventId != events.CgroupMkdir &&
+					eventId != events.CgroupRmdir &&
+					eventId != events.SecuritySbMount {
```

Comment:
I still don't understand why this is needed

Reply:
The goal is to remove this. I'm going to test it right now.

Reply:
Okay, it got resolved down the road at some point. https://github.com/aquasecurity/tracee/pull/2238/checks